test/core/card.spec.js

```diff
@@ -0,0 +1,22 @@
+const expect = require('./../helper').expect;
+
+const Card = require('./../../src/core/card');
```

Comment:
Usually you don't put the .spec.js files in a different folder to avoid this: they should go in the same folder of the modules. Btw `./../` is the same as `../`